index.html

```diff
@@ -484,7 +484,8 @@ <h3 class="title">Download NewPipe</h3>
                             <div class="col-xs-12 col-sm-6 col-lg-4" id="download-stable">
                                 <h4>F-Droid</h4>
                                 <div class="download-info">
-                                    <p class="download-version api-f-droid-stable-version">0.11.6</p>
+									<!--This is the closest we get to the actual release on FDroids website-->
+                                    <p class="download-version api-f-droid-stable-version"><a href="https://f-droid.org/packages/org.schabi.newpipe/#screenshots">Latest version on F-Droid</a></p>
```

Comment:
Created https://gitlab.com/fdroid/fdroid-website/merge_requests/481 to solve this :)

Reply:
Correct URL is now: https://f-droid.org/packages/org.schabi.newpipe/#latest

Reply:
Unbelievable. This mad lad just did it. Will update accordingly.

Reply:
done